pkg/handler/handler.go

```diff
@@ -97,3 +200,20 @@ func (pbh *prowBucketHandler) handleLogRequest(resp http.ResponseWriter, r *http
 		log.Printf("failed to write data for %s: %v", bucketPath, err)
 	}
 }
+
+func contains(publicRepos []string, repo string) bool {
+	for _, s := range publicRepos {
+		if s == repo {
+			return true
+		}
+	}
+	return false
+}
+
+// isAuthenticated returns true if the user has a signed session cookie.
+func isAuthenticated(req *http.Request) bool {
+	if _, err := sessionStore.Get(req, sessionName); err == nil {
```

Comment:
Thats what I meant with the comment above, doesn't this imply `sessioName` has to be unique per user?

Reply:
Replied above.